tests/tests.rs

```diff
@@ -0,0 +1,200 @@
+use std::rc::Rc;
+
+use candid::{CandidType, Decode, Encode, Nat};
+use evm_rpc::*;
+use ic_base_types::{CanisterId, PrincipalId};
+use ic_ic00_types::BoundedVec;
+use ic_state_machine_tests::{CanisterSettingsArgs, StateMachine, StateMachineBuilder, WasmResult};
+use ic_test_utilities_load_wasm::load_wasm;
+use serde::de::DeserializeOwned;
+
+const DEFAULT_CALLER_TEST_ID: u64 = 10352385;
+const DEFAULT_CONTROLLER_TEST_ID: u64 = 10352386;
```

Comment:
Is there any meaning attached to these IDs? (In other words, why not simpler/shorter IDs?)

Reply:
`10352385` comes from [this MR](https://gitlab.com/dfinity-lab/public/ic/-/blob/3ace016d71ce1bcc94ee39d1959aaa14728002dd/rs/ethereum/cketh/minter/tests/tests.rs#L38). @gregorydemay, do you know if there was any importance to the original ID?

Reply:
@rvanasa : The concrete value is not relevant. In ckETH a deposit needs to be linked to an IC principal and we use for this `DEFAULT_PRINCIPAL_ID` which has value `10352385`.